src/useFormState.js

```diff
@@ -33,13 +34,16 @@ export default function useFormState(initialState, options) {
   const createPropsGetter = type => (...args) => {
     const { name, ownValue, ...inputOptions } = parseInputArgs(args);
 
-    const hasOwnValue = !!toString(ownValue);
-    const hasValueInState = formState.current.values[name] !== undefined;
     const isCheckbox = type === CHECKBOX;
     const isRadio = type === RADIO;
     const isSelectMultiple = type === SELECT_MULTIPLE;
+    const isRaw = type === RAW;
+    const hasOwnValue = isRaw ? !!ownValue : !!toString(ownValue);
+    const hasValueInState = formState.current.values[name] !== undefined;
 
-    const key = `${type}.${name}.${toString(ownValue)}`;
+    const key = `${type}.${name}.${
+      isRaw ? JSON.stringify(ownValue) : toString(ownValue)
```

Comment:
The reason I changed `key` was because this is being used as a key in a cache, and `{foo: 'bar'}` and `{foo: 'baz'}` are clearly different values, but `toString()` is going to turn both of them into '[object Object]'.  Although, I didn't look *too* hard into what we were caching, so maybe this isn't a problem?  :P

Reply:
> Although, I didn't look too hard into what we were caching, so maybe this isn't a problem? :P

Key is a unique identifier I'm use to cache certain input props that shouldn't change across re-rerenders. For each input there are 3 values that guarantee its uniqueness: type, name, and ownValue. 

ownValue was only necessary for checkboxes - we could have multiple checkboxes with the same `name` but different `ownValue`s. **I highly doubt that there will be multiple `raw` inputs with the same name but different ownValues that are objects** (but I could be wrong). 

With `[object Object]`, worst case scenario, we're going to return the same cached `onChange`/`onBlur` for all raw inputs with ownValue as an object.